jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gke.yaml

```diff
@@ -541,9 +541,10 @@
             job-env: |
                 export E2E_OPT="--check_version_skew=false"
                 export E2E_UPGRADE_TEST="true"
-                export GINKGO_UPGRADE_TEST_ARGS="--ginkgo.focus=\[Feature:MasterUpgrade\] --upgrade-target=ci/latest-{version-new}"
+                export GINKGO_UPGRADE_TEST_ARGS="--ginkgo.focus=\[Feature:MasterUpgrade\] --upgrade-target=ci/latest-{version-new} --upgrade-image=container_vm"
```

Comment:
Just to be safe here, remove it from the old jobs, we plan on removing these jobs in a couple of weeks anyways.
i.e. "--upgrade-image=container_vm"